syntax = "proto3";

package bookstore;

import "order.proto";

service TransactionService {
    rpc InitVerifyTransaction (OrderData) returns (EmptyMessage);
    rpc VerifyItems (OrderInfo) returns (OrderResponse);
    rpc VerifyUserData (OrderInfo) returns (OrderResponse);
    rpc VerifyCreditCard (OrderInfo) returns (OrderResponse);
    rpc ClearData (OrderInfo) returns (ClearDataResponse);
}
